doc-changelog/action.yml

```diff
@@ -0,0 +1,138 @@
+name: >
+  Documentation Changelog
+
+description: |
+  Adds a newsfragment to the target repository using ``towncrier``
+  containing the title and number of the pull request.
+
+
+inputs:
+
+  # Required inputs
+
+  path-to-fragment:
+    description: >
+      Path to where the towncrier fragment is stored
+    required: true
+    type: string
+
+  pr-branch:
+    description: >
+      Name of the pull request
+    required: true
+    type: string
```

Comment:
That's a good point, @germa89. Could we check if the environment gets shared with the reused actions, @klmcadams? If so, we could skip these inputs.

Reply:
@germa89 @jorgepiloto Oh yeah, I must have forgotten to make a new description. The description should be the name of the PR branch. The reason I had those inputs was because I didn't know that you could do ${{ github... }} within the action, and that it would refer to the context of the branch it was being run on

This is the ci_cd file I was using to test the action: https://github.com/ansys-internal/actions-sandbox/blob/15e61871243b198706aae00c8eb4bf9dd90bd6db/.github/workflows/ci_cd.yml#L29

I think it would be nice if this only ran on PRs when they are first created, and any time the title is changed. I'm not sure if there's a title change event but I can do more research. Is there a way to specify within the action to only run in certain scenarios?